src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -1054,7 +1060,7 @@ def _get_rand_attn_plan(from_seq_length, from_block_size, num_rand_blocks):
 
     @staticmethod
     def _bigbird_block_rand_mask(
-        from_seq_length, to_seq_length, from_block_size, to_block_size, num_rand_blocks, last_idx=-1
+        from_seq_length, to_seq_length, from_block_size, to_block_size, num_rand_blocks, deterministic, last_idx=-1
```

Comment:
Wonder if it's better to make this a non-static method? E.g. so that we can do `self.training` from within the method and not have to pass `deterministic` as an argument? Just a suggestion, fine either way for me.

Reply:
@sanchit-gandhi yeah I was thinking about it and that is why I asked about `@staticmethod` for this method :)